### PR TITLE
Replace node-glob by fast-glob

### PR DESCRIPTION
**Pull request by pvdlg**
Per [comment](https://github.com/sindresorhus/globby/pull/68#issuecomment-362805086)

### CODE REVIEW

File: gitignore.js [hunk 1]
```diff
@@ -1,12 +1,19 @@
 'use strict';
 const fs = require('fs');
 const path = require('path');
-const glob = require('glob');
+const fg = require('fast-glob');
```
Comment (kevva) [inline gitignore.js]: Name it `fastGlob` so that it's easier to understand what it is right away.

**Comment by kevva** [comment]
Fantastic :+1:

File: gitignore.js [hunk 2]
```diff
@@ -72,7 +79,7 @@ const normalizeOpts = opts => {
 module.exports = o => {
 	const opts = normalizeOpts(o);
 
-	return globP('**/.gitignore', {ignore: opts.ignore, cwd: opts.cwd})
+	return fg('**/.gitignore', {ignore: opts.ignore.concat(DEFAULT_IGNORE), cwd: opts.cwd})
```
Comment (kevva) [inline gitignore.js]: Do we want to be able to overwrite the default ones? I guess putting `DEFAULT_IGNORE` at the end would take precedence over the users?

File: index.js [hunk 3]
```diff
@@ -1,11 +1,10 @@
 'use strict';
 const arrayUnion = require('array-union');
 const glob = require('glob');
-const pify = require('pify');
+const fg = require('fast-glob');
```
Comment (kevva) [inline index.js]: See comment above.

**Comment by pvdlg** [comment]
@kevva thanks for the review. I made the changes you suggested.

**Comment by kevva** [comment]
Also, I think [these](https://github.com/pvdlg/globby/blob/100a0906f6bf66830c07c60bd1aa100dd8b4f51a/index.js#L25-L28) could be removed since `fast-glob` doesn't seem to support them.

**Comment by sindresorhus** [comment]
@pvdlg Yay! A new `globby` version is out. Would you be able to update `globby` in XO?

**Comment by pvdlg** [comment]
yep, I'm on it!

**Pull request merged**